src/compile/axis.ts

```diff
@@ -61,6 +61,19 @@ export function parseInnerAxis(channel: Channel, model: Model): VgAxis {
     }
   });
 
+  const props = model.axis(channel).properties || {};
+
+  // it might have more to be included here.
+  ['grid'].forEach(function(group) {
+    const value = properties[group] ?
```

Comment:
I am not sure about if there's more properties needed to be added here so I leave here as a list.